src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -1629,7 +1645,7 @@ char *wm_vuldet_report_agent_vulnerabilities(sqlite3 *db, agent_software *agents
 
         if (agents_it->dist == FEED_WIN) {
             if (wm_vuldet_win_nvd_vulnerabilities(db, agents_it, flags)) {
-                return agents_it->agent_id;
+                return -1;
```

Comment:
Use `OS_INVALID` here and below

Reply:
Done https://github.com/wazuh/wazuh/pull/5015/commits/d902c9a8950c5413d370ec65893698e2487887c8